router.php

```diff
@@ -54,60 +64,72 @@ function _wp_filter_build_unique_id($tag, $function, $priority) {
 
 			return $obj_idx;
 		}
-	} else if ( is_string($function[0]) ) {
+	} elseif ( is_string( $function[0] ) ) {
 		// Static Calling
 		return $function[0] . '::' . $function[1];
 	}
 }
 
 function _get_full_host( $url ) {
+  // phpcs:ignore WordPress.WP.AlternativeFunctions.parse_url_parse_url
```

Comment:
Indentation is off here.
```suggestion
	// phpcs:ignore WordPress.WP.AlternativeFunctions.parse_url_parse_url
```